.github/workflows/test.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       with:
         version: v1.29
         args: --timeout=600s
+    - name: Format
+      run: if [ "$(gofmt -l . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
What does the output look like if this fails?

Reply:
If the condition hits it just fails as exit code 1 but its the only thing run there. I realize you are right i should output the files that need formatting.